README.md

```diff
@@ -72,6 +72,9 @@ The guidelines for setting up and running the tests are mentioned in the [testin
   - height: Height of the page on which badges (in groups of 8) will be printed. Value should be in mm.
             For example: **"420mm"**.
 
+- A set of sample csv files can be found [here](https://github.com/fossasia/badgeyay/tree/development/v1/sample).
```

Comment:
Change this to not use `v1` it will be deleted

Reply:
Okay sure. Should I use the ones in `backend`?